python/ql/examples/qlpack.yml

```diff
@@ -1,3 +1,4 @@
-name: codeql-python-examples
-version: 0.0.0
-libraryPathDependencies: codeql-python
+name: codeql/python-examples
+version: 0.0.1
+dependencies:
+  codeql/python-queries: 0.0.1
```

Comment:
```suggestion
  codeql/python-all: 0.0.1
```

Reply:
That shouldn't be necessary since `codeql/python-queries` already depends on `codeql/python-all`. All imported dependencies are re-exported to dependents. There is currently no way to have a private dependency that is invisible to downstream consumers.

Reply:
I mean that the example pack shouldn't depend on the queries. The example pack show how to use the library.

Reply:
Ah...yes. That makes sense. Thanks for suggesting.